com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -839,6 +839,42 @@ internal ComputeBuffer GetHistogramBuffer()
             return m_HistogramBuffer;
         }
 
+        internal void ComputeProceduralMeteringParams(HDCamera camera, out Vector4 proceduralParams1, out Vector4 proceduralParams2)
+        {
+            Vector2 proceduralCenter = m_Exposure.proceduralCenter.value;
+            if (camera.exposureTarget != null && m_Exposure.centerAroundExposureTarget.value)
+            {
+                var transform = camera.exposureTarget.transform;
+                // Transform in screen space
+                Vector3 targetLocation = transform.position;
+                if (ShaderConfig.s_CameraRelativeRendering != 0)
+                {
+                    targetLocation -= camera.camera.transform.position;
+                }
+                var ndcLoc = camera.mainViewConstants.viewProjMatrix * (targetLocation);
+                ndcLoc.x /= ndcLoc.w;
+                ndcLoc.y /= ndcLoc.w;
+
+                Vector2 targetUV = new Vector2(ndcLoc.x, ndcLoc.y) * 0.5f + new Vector2(0.5f, 0.5f);
+                targetUV.y = 1.0f - targetUV.y;
```

Comment:
QA: please be sure to test Target on both windows on Metal,  just to be sure it work on all platform